### ISSUE TITLE
alter table to add a UDT column crashes yb-tserver my dev machine

### ISSUE THREAD

**Issue by nickhristov** [YugaByte/yugabyte-db#10]
So, I did an ALTER statement on a table, with a custom type.

Here is the short video of the scenario: https://asciinema.org/a/UVQcqu0ZFXvUoWNcEwxdNuj9x

Here is the log:

```
E1221 17:18:48.937922 196403200 process_context.cc:180] SQL Error: No Namespace Used
CREATE TYPE dagger ( junk DOUBLE );
            ^^^^^^
*** Aborted at 1513905573 (unix time) try "date -d @1513905573" if you are using GNU date ***
PC: @        0x108581084 yb::QLType::ToQLTypePB()
*** SIGSEGV (@0x8) received by PID 90780 (TID 0x70000b942000) stack trace: ***
    @     0x7fff579f0f5a _sigtramp
    @        0x10ec9a088 (unknown)
    @        0x108577a07 yb::ColumnSchemaToPB()
    @        0x107468df8 yb::client::YBTableAlterer::Data::ToRequest()
    @        0x107414f94 yb::client::YBTableAlterer::Alter()
    @        0x106bec73f yb::ql::Executor::ExecPTNode()
    @        0x106beb79c yb::ql::Executor::ExecTreeNode()
    @        0x106beae96 yb::ql::Executor::Execute()
    @        0x106bead38 yb::ql::Executor::ExecuteAsync()
    @        0x106b99f84 yb::ql::QLProcessor::RunAsync()
    @        0x106786ad2 yb::cqlserver::CQLProcessor::ProcessRequest()
    @        0x106785c05 yb::cqlserver::CQLProcessor::ProcessRequest()
    @        0x1067854a0 yb::cqlserver::CQLProcessor::ProcessCall()
    @        0x1067a1602 yb::cqlserver::CQLServiceImpl::Handle()
    @        0x108371218 yb::rpc::ServicePoolImpl::Handle()
    @        0x108371bc3 yb::rpc::TasksPool<>::WrappedTask::Run()
    @        0x108374e60 yb::rpc::(anonymous namespace)::Worker::Execute()
    @        0x10892c0ea yb::Thread::SuperviseThread()
    @     0x7fff579fa6c1 _pthread_body
    @     0x7fff579fa56d _pthread_start
    @     0x7fff579f9c5d thread_start
```

**Comment by kmuthukk** [comment]
Thanks for posting the detailed steps @nickhristov . Looks like ALTER TABLE to add a UDT (TYPE) column isn't working.

CC: @robertpang , @m-iancu: Here's a small sample to repro.

create keyspace if not exists test;
use test;

drop table if exists heart;
drop type if exists dagger;

create type dagger (type text, price double);
create table heart (id int, name text, PRIMARY KEY (id));

-- add of a primitive type column work.
alter table heart add zipcode int;

describe heart;
-- adding a UDT column as part of an alter table fails with the above stack trace.
alter table heart add d dagger;

**Comment by praneeth-nimmagadda** [comment]
Weel yeah, yb-tserver crashes on my dev machine when I some times run a create table and also gives ....

:NoHostAvailable.....statement....

**Comment by m-iancu** [comment]
Fixed by cb56881.

**Comment by SergeyPotachev** [comment]
Please disregard commit ce47c60 mistakenly added. It is not related to this issue.
